### ISSUE TITLE
Vocalization of the buttons camera / sound

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#1043]
## Bug Report

**Problematic behavior**
When I use keyboard shortcuts to turn the camera and microphone on and off, I do get a visual state change but it is not announced by screen readers.

**Expected behavior/code**
When I activate these keyboard shortcuts or even with the VO + space shortcut, there must be a vocal announcement of the state change: for example: the microphone is muted / the camera is disabled.

**Environment**
- Meet version: Macos X
- Platform: Chrome / Safari
